redis/1.9/README.md

```diff
@@ -0,0 +1,138 @@
+- [Access the Redis instance](##access-the-redis-instance)
+- [Uninstall mr-redis](#uninstall-mr-redis)
+
+## Prerequisites
+
+- A running DC/OS 1.9 cluster with at least 2 nodes with 1 CPU and 128MB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.9/usage/cli/install/) installed.
+
+## Single container redis
+Note, that this service is targeted at testing redis and not for production workloads!
+You can install the single redis container with a simple `dcos package install redis`.
+Then the easiest option to test redis is by storing a key manually via the redis-cli
+- SSH into node where redis is running: 
+``` bash 
+dcos node ssh --master-proxy --mesos-id=$(dcos task redis --json | jq -r '.[] | .slave_id')
+```
+NOTE: This requires you to have the ssh-key required to conenct to the machines added to your local ssh agent (e.g., via ssh-add my_public_key).
+- Because redis is running in docker container, we can list all docker containers using docker ps and get the ContainerID.
+- Connect to a bash session to the running container: `sudo docker exec -i -t CONTAINER_ID /bin/bash`
+- Start the redis CLI: redis-cli
+  - Set key i`set mykey key1`
+  - Check value is there `get mykey`
+
+##  mr-redis
+The DC/OS service [mr-redis](https://github.com/mesos/mr-redis), maintained by [Huawei](http://www.huawei.com/en/)
+is a Mesos framework allowing you to manage Redis datastores standalone or in a clustered setup.
+
+## Install mr-redis
+
+To install mr-redis from the DC/OS CLI, do the following:
+
+```bash
+$ dcos package install mr-redis
+In order for redis framework to start successfully it requires atleast 1 CPU and 128MB of RAM including ports.
+Note that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.
+Continue installing? [yes/no] yes
+Installing Marathon app for package [mr-redis] version [0.0.1]
+Once the cluster initializes download cli from https://github.com/mesos/mr-redis/releases/download/v0.01-alpha/mrr and follow the instructions in github.com/mesos/mr-redis README on how to initialize the cli, you could also use the REST api's directly to create redis instances
+```
+To validate that the mr-redis service is running and healthy you can go to the DC/OS UI:
+
+![Services](img/services1.9.png)
+
+Note that the mr-redis framework scheduler is serving on `mrredis.mesos:5656`.
+
+## Create a Redis instance
+
+While the mr-redis service we installed in the previous step is capable of supervising Redis instances, you will need to download the `mrr`, the mr-redis CLI to create and delete Redis instances. So, let's install the `mrr` (from within the DC/OS cluster):
+
+```bash
+$ dcos node ssh --master-proxy --leader
+
+core@ip-10-0-6-55 ~ $ curl -s -L https://github.com/mesos/mr-redis/releases/download/v0.01-alpha/mrr_linux_amd64 -o mrr
+core@ip-10-0-6-55 ~ $ chmod +x mrr
+core@ip-10-0-6-55 ~ $ ./mrr init http://mrredis.mesos:5656
```

Comment:
I got an error here, did you?

$ ./mrr init http://mrredis.mesos:5656
Cli is not initalized err=open /tmp/.MrRedis: no such file or directory
$mrr init <http://MrRedisEndPoint>

Running it a second time does not present the error and /tmp/.MrRedis exists. Probably not worth doing anything about but I am mentioning this for posterity in case anyone wants to have a look.